src/backend/gporca/data/dxl/minidump/ReplicatedTableCTE.mdp

```diff
@@ -1443,7 +1443,7 @@
         </dxl:LogicalJoin>
       </dxl:LogicalCTEAnchor>
     </dxl:Query>
-    <dxl:Plan Id="0" SpaceSize="8">
+    <dxl:Plan Id="0" SpaceSize="7">
```

Comment:
it is interesting that in this case CPhysicalMotionRandom is added in Group 1 but Number of plan alternatives decreases